driver-core/src/test/java/com/datastax/driver/core/IndexMetadataTest.java

```diff
@@ -102,6 +103,13 @@ public void beforeClass() {
         return builder.build();
     }
 
+    @Test(groups = "unit")
+    public void should_use_all_fields_in_equals_and_hashCode() {
+        EqualsVerifier.forClass(IndexMetadata.class)
+            .allFieldsShouldBeUsedExcept("table")
```

Comment:
The parent table name is not considered in equals/hashCode.  Since this comparison is only done between Index changes in the same table i'm guessing this is not needed?